src/modules/dataGroup.js

```diff
@@ -0,0 +1,160 @@
+import _ from 'lodash';
+import ClassNames from 'classnames';
+import Icon from '../elements/icon';
+import PropTypes from 'prop-types';
+import React from 'react';
+import Header from '../elements/header.js';
+import List from '../elements/list.js';
+import Utils from '../utils/utils.js'
+
+class DataGroupRow extends React.PureComponent {
```

Comment:
Candidate for a functional style component...?
Please note that multiple class components per file is no longer permitted by the newer ESLINT configurations.

Reply:
@groberts314 Thanks, I will split those on seperate files. Just waiting for other reviews and will start working on it

Reply:
Sure thing, fair enough.  Thank you sir! =)

Reply:
Fixed splitting multiple class components into separate files. Let's have a discussion over functional style component and `React.PureComponent`

Reply:
> Let's have a discussion over functional style component and `React.PureComponent`

Yes, agreed.  @morethanfire @IlyaRadinsky ... what are your gents' thoughts on this matter?